kiwi/xml_parse.py

```diff
@@ -3516,12 +3506,9 @@ def export(self, outfile, level, namespace_='', name_='user', namespacedef_='',
         else:
             outfile.write('/>%s' % (eol_, ))
     def exportAttributes(self, outfile, level, already_processed, namespace_='', name_='user'):
-        if self.group is not None and 'group' not in already_processed:
-            already_processed.add('group')
-            outfile.write(' group=%s' % (quote_attrib(self.group), ))
-        if self.secgroups is not None and 'secgroups' not in already_processed:
-            already_processed.add('secgroups')
-            outfile.write(' secgroups=%s' % (quote_attrib(self.secgroups), ))
+        if self.groups is not None and 'groups' not in already_processed:
```

Comment:
Again, if your `user.get_groups()` would return a list or None, the first check would not be needed. Note, here is a bug: if `self.groups` is an empty string or an empty list, you are in a positive state!